examples/stickysession/nginx/sticky-ingress.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: nginx-test
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    ingress.kubernetes.io/sticky-enabled: "true"
+    ingress.kubernetes.io/sticky-name: "route"
+    ingress.kubernetes.io/sticky-hash: "sha1"
```

Comment:
The module only implements md5, sha1 and index. I've selected in this example sha1 to avoid collisions. But the default behaviour (also in ingress code) is to use md5 when nothing is selected.